projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx

```diff
@@ -113,9 +132,33 @@ export class DotFormComponent {
 
     private handleSubmit(event: Event): void {
         event.preventDefault();
-        this.onSubmit.emit({
-            ...this.value
-        });
+
+        fetch(SUBMIT_FORM_API_URL, {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({
+                stName: this.variable || 'ContactUs',
```

Comment:
why `ContactUS`

Reply:
No... this was just for testing.. needs to remove.